src/app/components/AddDocumentForm/AddDocumentForm.module.css

```diff
@@ -14,3 +14,14 @@
   color: var(--color-text);
   background-color: var(--color-tertiary);
 }
+
+.documentSave:hover {
+  cursor: pointer;
+}
+
+.documentSave:disabled,
+button[disabled] {
+  cursor: default;
```

Comment:
Is it necessary to set the cursor to default here?

Reply:
Thanks! Yes, it's necessary if you want that effect while the button is disabled.